server/plugin_test.go

```diff
@@ -142,6 +142,15 @@ func TestPlugin(t *testing.T) {
 			api.On("KVSetWithExpiry", fmt.Sprintf("%v%v", postMeetingKey, 123), mock.AnythingOfType("[]uint8"), mock.AnythingOfType("int64")).Return(nil)
 			api.On("KVSetWithExpiry", "zoomuserstate_theuserid", mock.AnythingOfType("[]uint8"), mock.AnythingOfType("int64")).Return(nil)
 
+			api.On("KVSetWithOptions", "mutex_mmi_bot_ensure", mock.AnythingOfType("[]uint8"), model.PluginKVSetOptions{Atomic: true, OldValue: []uint8(nil), ExpireInSeconds: 15}).Return(true, nil)
+			api.On("KVSetWithOptions", "mutex_mmi_bot_ensure", []byte(nil), model.PluginKVSetOptions{ExpireInSeconds: 0}).Return(true, nil)
+
+			api.On("EnsureBotUser", &model.Bot{
+				Username:    botUserName,
+				DisplayName: botDisplayName,
+				Description: botDescription,
+			}).Return(botUserID, nil)
```

Comment:
Interesting

Reply:
@mickmister What risk might you be talking about on this?